src/cli/Secrets.ts

```diff
@@ -1,34 +1,47 @@
+import process from 'process';
 import commander from 'commander';
-import { actionRunner, pkLogger, PKMessageType, determineNodePath } from '.';
+import { spawn, ChildProcess, exec } from 'child_process';
 import { PolykeyAgent } from '../lib/Polykey';
+import { actionRunner, pkLogger, PKMessageType, determineNodePath } from '.';
+
+
+const pathRegex = /^([a-zA-Z0-9_-]+)(?::)([a-zA-Z0-9_:-]+)(?:=)?([a-zA-Z_][a-zA-Z0-9_]+)?$/
```

Comment:
Does this regex parse all kinds of unix paths? Remember that unix paths allow spaces and newlines as well.

Reply:
I agree, this needs some work

Reply:
https://github.com/MatrixAI/js-polykey/issues/81